src/main/java/main/view/BaseServlet.java

```diff
@@ -139,15 +148,22 @@ protected void setErrorHeader(@NotNull HttpServletResponse resp, String errorMes
     }
 
     @Nullable
-    private String getSessionId(@NotNull HttpServletRequest req){
+    private String getSessionId(@NotNull HttpServletRequest req) throws AqualityException {
         String header = req.getHeader("Authorization");
         if(header != null){
+            validateAuthHeader(header);
             String[] strings = header.split(" ");
             return strings[1];
         }
         return null;
```

Comment:
is it make sense to return null instead of throwing an exception?

Reply:
fixed